earth2studio/data/xr.py

```diff
@@ -169,3 +169,114 @@ def __call__(
             arrs.append(self.das[yr][mon].sel(time=tt, variable=variable))
 
         return xr.concat(arrs, dim="time")
+
+
+class DataArrayPathList:
```

Comment:
Can you add some unit tests for this in the test_xr.py

https://github.com/NVIDIA/earth2studio/blob/main/test/data/test_xr.py